src/test/java/uk/gov/justice/digital/hmpps/cmd/api/repository/ShiftNotificationRepositoryTest.kt

```diff
@@ -72,6 +72,19 @@ class ShiftNotificationRepositoryTest(
                     now.plusDays(1).atStartOfDay())
             assertThat(notifications).isEmpty()
         }
+
+        @Test
+        fun `Should return count of matching`() {
+            val date = now.plusDays(3).atStartOfDay()
+            val notification = getValidShiftNotification(date, date)
+            repository.save(notification)
+
+            val notifications = repository.countAllByQuantumIdAndShiftDateAndShiftType(
+                    "XYZ",
+                    date,
+                    "SHIFT")
+            assertThat(notifications).isEqualTo(1)
+        }
```

Comment:
Could you do 0 and 2 please? 
And a test where you put two different instances with he same values in them, and show how many come out.

Reply:
Added 0 and 2. The test with 2 values needs to have the same details as that's what we're matching on